generated/nidaqmx/nidaqmx_service.h

```diff
@@ -21,7 +21,7 @@
 
 namespace nidaqmx_grpc {
 
-class NiDAQmxService final : public NiDAQmx::Service {
+class NiDAQmxService final : public NiDAQmx::ExperimentalWithCallbackMethod_RegisterDoneEvent<NiDAQmx::Service> {
```

Comment:
Note: These have been "de-experimentalized" in the latest grpc release. I've started #281 which will pull this in. But I would like to resolve the [AMD build issues](https://bugs.chromium.org/p/boringssl/issues/detail?id=423) first.